website_event_crm/models/crm_lead.py

```diff
@@ -65,6 +48,47 @@ def _cron_auto_invite_website_event_type(self):
             except Exception:
                 _logger.exception("Failure trying to invite to website event type.")
 
+    @api.model
+    def _cron_auto_invite_website_event_type(self):
```

Comment:
Can we avoid to reorder this for reducing the diff?

Reply:
Done